docs/source/examples/checks/methodology/single_feature_contribution_train_test.ipynb

```diff
@@ -5,123 +5,83 @@
    "id": "21c5d5bc",
```

Comment:
<p>Add a paragraph after the result saying that it caught the changes we introduced?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/642/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>+1</p>